ega-charts/localega/templates/filedatabase-deploy.yaml

```diff
@@ -31,7 +31,7 @@ spec:
         - name: SPRING_PROFILES_ACTIVE
           value: no-oss
         - name: DB_SCHEMA
-          value: {{ .Values.config.postgres_db_schema | quote }}
+          value: "local_ega"
```

Comment:
Is there a reason why we fix this. I think we should let it configurable, even though it might not change, I think some people might have an existing DB that they would use.

if we keep like this, remove from README.md as well

Reply:
fixed